doc/s9s.dbversions.1

```diff
@@ -0,0 +1,69 @@
+.TH S9S-DBVERSIONS 1 "December 19, 2023"
```

Comment:
Did you do the check of spelling in the document? If not yet please do it. Thank you.